frontend/src/pages/LoginPage.jsx

```diff
@@ -14,13 +14,18 @@ export default function LoginPage() {
   };
 
   return (
-    <div>
-      <form onSubmit={handleSubmit}>
-        <input value={username} onChange={e => setUsername(e.target.value)} type="text" />
-        <input value={password} onChange={e => setPassword(e.target.value)} type="password" />
-        <button type="submit">Login</button>
-      </form>
-      <button onClick={() => navigate("/auth/register")}>Sign up</button>
+    <div className={css.mainContainer}>
+      <div className={css.formBlock}>
+        <h1>AUTHORIZATION</h1>
```

Comment:
use `text-transform: uppercase;` css property to make uppercase text, instead of using capitalized text in the code. I don't like caps :)